src/main/java/io/jenkins/plugins/audit/plugins/CustomHTMLLayout.java

```diff
@@ -14,15 +14,14 @@
  * See the license for the specific language governing permissions and
  * limitations under the license.
  */
-package io.jenkins.plugins.audit;
+package io.jenkins.plugins.audit.plugins;
```

Comment:
This move / renaming looks odd.

Reply:
The log4j config adds a package to scan for plugins, so to save startup time, I moved the custom plugins into their own package so it has to scan less classes.